content/enterprise_influxdb/v1.8/guides/hardware_sizing.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Hardware sizing guidelines 
+Description: >
+  Review configuration and hardware guidelines for InfluxDB OSS (open source) and InfluxDB Enterprise.
+menu:
+  enterprise_influxdb_1_8:
+    weight: 40
+    parent: Guides
+---
+
+Review configuration and hardware guidelines for InfluxDB Enterprise:
+
+* [Enterprise overview](#enterprise-overview)
+* [Query guidelines](#query-guidelines)
+* [InfluxDB Enterprise cluster guidelines](#influxdb-enterprise-cluster-guidelines)
+* [When do I need more RAM?](#when-do-i-need-more-ram)
+* [Recommended cluster configurations](#recommended-cluster-configurations)
+* [Storage: type, amount, and configuration](#storage-type-amount-and-configuration)
+
+For OSS hardware sizing guidelines, click on following [LINK](https://docs.influxdata.com/influxdb/v1.8/guides/hardware_sizing/).
+
+> **Disclaimer:** Your numbers may vary from recommended guidelines. Guidelines provide estimated benchmarks for implementing the most performant system for your business.
+
+## Enterprise overview
+
+InfluxDB Enterpise supports the following:
+
+- more than 750,000 field writes per second
+- more than 100 moderate queries per second ([see Query guides](#query-guidelines))
+- more than 10,000,000 [series cardinality](/influxdb/v1.8/concepts/glossary/#series-cardinality)
+
+InfluxDB Enterprise distributes multiple copies of your data across a cluster,
+providing high-availability and redundancy, so an unavailable node doesn’t significantly impact the cluster.
+Please [contact us](https://www.influxdata.com/contact-sales/) for assistance tuning your system.
+
+If you want a single node instance of InfluxDB that's fully open source, requires fewer writes, queries, and unique series than listed above, and do **not require** redundancy, we recommend InfluxDB OSS.
+
+> **Note:** Without the redundancy of a cluster, writes and queries fail immediately when a server is unavailable.
+
+## Query guidelines
+
+> Query complexity varies widely on system impact. Recommendations for both single nodes and clusters are based on **moderate** query loads.
+
+For **simple** or **complex** queries, we recommend testing and adjusting the suggested requirements as needed. Query complexity is defined by the following criteria:
+
+| Query complexity | Criteria                                                                              |
+|------------------|---------------------------------------------------------------------------------------|
```

Comment:
```suggestion
|:------------------|:---------------------------------------------------------------------------------------|
```

Reply:
let's check table alignment in all hardware sizing docs to be sure all column headers are left aligned